### PR TITLE
CAN naming consistency

### PR DESCRIPTION
**Pull request by ajaku**
CAN messages and signals have been renamed to improve consistency.

### CODE REVIEW

File: can/can.yml [hunk 1]
```diff
@@ -1,206 +1,200 @@
...
+    - THROTTLE
+    - BRAKE
+    - ENCF
+    - ENCR
+    - PB
+    - WHL
 
 messages:
-- dbwNode_Status:
+- DBW_NodeStatus:
```
Comment (jacobkoziej) [inline can/can.yml:12]: `NodeStatus`

File: can/can.yml [hunk 2]
```diff
@@ -1,206 +1,200 @@
...
     id: 0xE0
     cycletime: 100
 
     template: dbwnodes
 
     signals:
-      - SystemStatus:
+      - systemStatus:
```
Comment (jacobkoziej) [inline can/can.yml:19]: `sysStatus`

File: can/can.yml [hunk 3]
```diff
@@ -1,206 +1,200 @@
...
             - POWERON
             - WATCHDOG_RESET
             - UNKNOWN
-      - Esp32ResetReasonCode:
+      - esp32ResetReasonCode:
           description: ESP32 reset reason code (enum RESET_REASON)
           width: 5
 
-- dbwNode_Info:
+- DBW_NodeInfo:
```
Comment (jacobkoziej) [inline can/can.yml:41]: `NodeInfo`

File: can/can.yml [hunk 4]
```diff
@@ -1,206 +1,200 @@
...
             width: 7
-        - BrakePercent:
+        - brakePercent:
             description: Brake percentage.
             maximum: 100
             unit: percent
             width: 7
 
-- dbwNode_Steering_Cmd:
+- DBW_NodeSteeringCmd:
```
Comment (jacobkoziej) [inline can/can.yml:179]: `WHL_SteeringCmd`

File: can/can.yml [hunk 5]
```diff
@@ -209,26 +203,26 @@ messages:
...
             width: 40
 
-- dbwESTOP:
+- DBW_ESTOP:
     id: 0x41F
     cycletime: 100
 
     signals:
-      - Source:
+      - source:
```
Comment (jacobkoziej) [inline can/can.yml:218]: `src`

File: can/can.yml [hunk 6]
```diff
@@ -237,73 +231,73 @@ messages:
...
+      - filler1:
           description: Filler.
           width: 7
-      - Begin:
+      - begin:
           description: Begin the update.
           width: 1
 
-- dbwUpdater_Update_Data:
+- DBW_UpdaterUpdateData:
```
Comment (jacobkoziej) [inline can/can.yml:269]: `UpdateData`

File: can/can.yml [hunk 7]
```diff
@@ -237,73 +231,73 @@ messages:
...
+      - position:
           description: Index of 32-bit data chunk.
           width: 24
-      - Data:
+      - data:
           description: Image data.
           width: 40
 
-- dbwNode_Update_Response:
+- DBW_NodeUpdateResponse:
```
Comment (jacobkoziej) [inline can/can.yml:283]: `UpdateResponse`

File: can/can.yml [hunk 8]
```diff
@@ -237,73 +231,73 @@ messages:
...
     template: dbwnodes
 
     signals:
-      - Ready:
+      - ready:
           description: Ready to begin update.
           width: 1
 
-- dbwUpdater_Update_Done:
+- DBW_UpdaterUpdateDone:
```
Comment (jacobkoziej) [inline can/can.yml:294]: `UpdateDone`

**Review by jacobkoziej — CHANGES_REQUESTED** [review]
Just a few changes to be made, @dmezh would you mind checking on the suggested bootloader renames?

**Comment by ajaku** [comment]
All the reviews were implemented except for the Updater ones (still not sure what the deal is with those).

**Comment by dmezh** [comment]
@ajaku you can leave the updater messages as-is. Looks like you have a merge conflict in can.yml.

**Comment by dmezh** [comment]
@ajaku Ok, the updater message mess is mine to clean up.. can I push to your branch?

**Review by jacobkoziej — DISMISSED** [review]

File: can/can.yml [hunk 9]
```diff
@@ -1,211 +1,205 @@
...
-      - GitDirty:
-          description: Repository was dirty at build time.
-          width: 1
-      - EepromIdentity:
-          description: EEPROM identity.
-          width: 6
-
-- dbwNode_Accel_Data:
+      - gitHash:
+           description: Githash of the currently-running firmware.
```
Comment (dmezh) [inline can/can.yml:49]: Extra spaces?

File: can/can.yml [hunk 10]
```diff
@@ -242,73 +236,73 @@ messages:
...
+      - position:
           description: Index of 32-bit data chunk.
           width: 24
-      - Data:
+      - data:
           description: Image data.
           width: 40
 
-- dbwNode_Update_Response:
+- DBW_NodeUpdateResponse:
```
Comment (dmezh) [inline can/can.yml:288]:
```suggestion
- NodeUpdateResponse:
```

It's ok, leave it for now. Will be reworked in bootloader PR

**Review by dmezh — DISMISSED** [review]
Approving under the condition that this PR is merged squashed.

File: can/can.yml (hunk 9 above)
Comment (jacobkoziej) [reply can/can.yml]: Fixed. Ran the file through `yamllint`.

**Review by jacobkoziej — APPROVED** [review]

**Comment by dmezh** [comment]
I didn't know you could approve PRs with your own commits in them 😶

**Review by dmezh — APPROVED** [review]

**Pull request merged**